bin/setup

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
There is already a setup rake here: https://github.com/Codeminer42/cm42-central/blob/upgrade_rails/lib/tasks/fulcrum.rake#L3

Also, I think that this is would be better placed in another PR (or commit)